src/lib/components/Instances/InstanceTableRow.svelte

```diff
@@ -51,33 +45,53 @@
 		contenteditable="plaintext-only"
 		class="box-border p-3"
 		bind:innerText={localInstanceInput}
-		onblur={() => changeInstance({ ...instance, input: localInstanceInput })}
+		onblur={() => {
```

Comment:
updated these so fine-grained reactivity works. Reassigning in parent caused whole thing to re-render and update.